dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/EndpointManager.java

```diff
@@ -44,7 +41,7 @@ public class EndpointManager {
     private final ConcurrentHashMap<Class<?>, ArrayList<Session>> endpointSessionMap = new ConcurrentHashMap<Class<?>, ArrayList<Session>>();
 
     // Map of  sessions with corresponding active HttpSession
-    private final ConcurrentHashMap<String, SessionExt> httpSessionMap = new ConcurrentHashMap<String, SessionExt>();
+    private final ConcurrentHashMap<String, ArrayList<Session>> httpSessionMap = new ConcurrentHashMap<String, ArrayList<Session>>();
```

Comment:
Changing from`SessionExt` to `Session` is ok here? I'm assuming we don't need to call anything in the concrete `SessionExt` class and we're fine with the `Session` API?

Reply:
I guess I could have kept it as SessionExt. I later use casting to ensure we call the right methods: ((SessionExt) session).

We can discuss more in our meeting.